mailpoet/assets/js/src/email-editor/engine/store/types.ts

```diff
@@ -29,6 +29,88 @@ export type EmailEditorSettings = EditorSettings & ExperimentalSettings;
 export type EmailTheme = {
   version?: number;
   styles?: EmailStyles;
+  // Ref: https://github.com/WordPress/gutenberg/blob/38d0a4351105e6ba4b72c4dcb90985305aacf921/packages/block-editor/src/components/global-styles/hooks.js#L24C7-L24C21
+  settings?: {
```

Comment:
I see a lot of booleans. I would expect many of those to be a string (e.g., `color. background`). Are those booleans correct?

Reply:
Some may be wrong, but `settings` is not the same as `styles`. Most of these determine if a field is supported. Thats likely why "link" is not being shown.

Reply:
Oh! Makes sense! I thought these were for styles. Now I see styles reuse EmailStyles 👍